packages/esm-service-queues-app/src/queue-table/queue-table-metrics.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import QueueTableMetricsCard from './queue-table-metrics-card.component';
+import styles from '../patient-queue-metrics/clinic-metrics.scss';
+
+function QueueTableMetrics() {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <div className={styles.cardContainer} data-testid="clinic-metrics">
```

Comment:
All of the below are incorrect in the sense that:
a) The headers all need to be translation strings if they are hard-coded (eg. Total Patients)
b) The cards that relate to specific statuses need to be done dynamically by iterating over the queue, getting it's allowed statuses, and then displaying a card based on that queue/status combination